packages/calypso-e2e/src/browser-helper.ts

```diff
@@ -85,6 +85,9 @@ export function getLaunchConfiguration( chromeVersion: string ): BrowserContextO
 	config.userAgent = userAgent;
 	// Explicitly resize captured video resolution to the viewport size.
 	config.recordVideo = { dir: os.tmpdir(), size: config.viewport as ViewportSize };
+	// Specify the user locale.
+	config.locale = getLocale();
```

Comment:
There is already a `BrowserHelper.getLocale` method.

See https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/browser-helper.ts#L35-L37.

Reply:
That's the same method that I'm using to set `config.locale` here. Specifying the `locale` option when [creating a new browser context](https://github.com/Automattic/wp-calypso/blob/5545cf60c5356b36ba80ca556d9b9dce3b1ec8e0/packages/calypso-e2e/src/browser-manager.ts#L41), affects the `Accept-Language` request header, which is needed in this case.

Reply:
Ah gotcha, thanks for clarifying. I should've expanded the context around this a bit more to see that you were working with the launch configuration.